itests/org.openhab.core.config.discovery.tests/itest.bndrun

```diff
@@ -45,6 +49,7 @@ Fragment-Host: org.openhab.core.config.discovery
 	tec.uom.se;version='[1.0.8,1.0.9)',\
 	ch.qos.logback.core;version='[1.2.3,1.2.4)',\
 	slf4j.api;version='[1.7.25,1.7.26)',\
-	ch.qos.logback.classic;version='[1.2.3,1.2.4)',\
 	net.bytebuddy.byte-buddy;version='[1.9.7,1.9.8)',\
-	net.bytebuddy.byte-buddy-agent;version='[1.9.7,1.9.8)'
\ No newline at end of file
+	net.bytebuddy.byte-buddy-agent;version='[1.9.7,1.9.8)',\
+	com.google.gson;version='[2.8.2,2.8.3)',\
+	slf4j.simple;version='[1.7.21,1.7.22)'
```

Comment:
Any idea why the plugin adds `slf4j.simple` in this PR and `ch.qos.logback.classic` in my PR (https://github.com/openhab/openhab-core/pull/647)? I didn't like that change and even less like the idea of it being switched depending on who generates these files. It's also changed in the other files.

It would be even nicer if we don't have to generate and commit these files and update them for every dependency change.

Reply:
I assume it has been caused by an unclean development environment.
I forget to remove other commits in front of building and resolving.
slf4j.simple should not be part of the available bundles at all.

The files should be committed if there is any change (sure, we should eliminate changes if not necessary).

Reply:
There are additional ones that could be cleaned up some time, but I assume now it is a little bit better.